src/Chapter3.hs

```diff
@@ -910,6 +981,21 @@ Implement instances of "Append" for the following types:
 class Append a where
     append :: a -> a -> a
 
+newtype Gold = Gold { goldAmount :: Int } deriving (Show)
+instance Append Gold where
+  append :: Gold -> Gold -> Gold
+  append a b = Gold (goldAmount a + goldAmount b)
+
+instance Append [a] where
+  append :: [a] -> [a] -> [a]
+  append la lb = la ++ lb
```

Comment:
You can eta-reduce the implementation of `append` for brevity. So you can see how easy it is to create instances :slightly_smiling_face: 
```suggestion
    append = (++)
```